### PR TITLE
Added .DS_Store to .gitignore

### PR DESCRIPTION
**Pull request by gtrabanco**
And deleted from repo.

Done with vim.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -59,3 +59,6 @@ typings/
 
 
 \.idea/
+
+# osx files
+.DS_S*
```
Comment (peoplenarthax) [inline .gitignore:64]: is there anything else than `.DS_Store` to ignore?

Comment (pagoru) [reply .gitignore:64]: If anyone uses vscode... maybe

Comment (gtrabanco) [reply .gitignore:64]: > If anyone uses vscode... maybe

😂

Comment (gtrabanco) [reply .gitignore:64]: > is there anything else than `.DS_Store` to ignore?

Don't think so

Comment (peoplenarthax) [reply .gitignore:64]: Can we then just be specific about `.DS_Store` ? I think is good to be specific in gitignore

Comment (gtrabanco) [reply .gitignore:64]: > Can we then just be specific about `.DS_Store` ? I think is good to be specific in gitignore

Probably yes, but not sure if its possible to find sometimes the same file with different case.

**Review by peoplenarthax — APPROVED** [review]
Let's just get this merge and iterate if it is problematic

**Review by pagoru — APPROVED** [review]

**Pull request merged**